jakarta-package-alignment-mappings/src/main/java/com/palantir/gradle/jakartapackagealignment/VersionMapping.java

```diff
@@ -17,32 +17,26 @@
 
 import org.apache.maven.artifact.versioning.ComparableVersion;
 
-final class VersionMapping {
-    private final String jakartaGroupId;
-    private final String jakartaArtifactId;
+public final class VersionMapping {
     private final ComparableVersion maxJakartaVersionWithJavaxNamespace;
-    private final String mappedJavaeeCoord;
+    private final MavenCoordinate jakartaCoord;
+    private final MavenCoordinate mappedJavaeeCoord;
 
     VersionMapping(MavenCoordinate jakartaCoord, MavenCoordinate javaeeCoord) {
-        this.jakartaGroupId = jakartaCoord.getGroupId();
-        this.jakartaArtifactId = jakartaCoord.getArtifactId();
         this.maxJakartaVersionWithJavaxNamespace = new ComparableVersion(jakartaCoord.getVersion());
-        this.mappedJavaeeCoord = javaeeCoord.toString();
-    }
-
-    String getJakartaGroupId() {
-        return jakartaGroupId;
-    }
-
-    String getJakartaArtifactId() {
-        return jakartaArtifactId;
+        this.jakartaCoord = jakartaCoord;
+        this.mappedJavaeeCoord = javaeeCoord;
     }
 
     ComparableVersion getMaxJakartaVersionWithJavaxNamespace() {
         return maxJakartaVersionWithJavaxNamespace;
     }
```

Comment:
Thoughts on removing this method entirely in favor of building `ComparableVersion` on an as-needed basis in VerssionMappings?

Reply:
yeah we can do that

Reply:
one downside is that it becomes less clear that this mapping could technically apply to a range of versions, but i think in practice there's only one Jakarta EE release with the old `javax` namespace, so it's really always a single version.